Automattic/sensei
# Add email to be sent to teacher when a student replies to a private message

Resolves https://github.com/Automattic/sensei/issues/6475

## Proposed Changes
- Adds email to be sent to teacher when a student replies to a private email

## Testing Instructions
- Enable the email post type
- Run the tool Sensei LMS -> Tools -> Re-Create Emails (Do it even if you have done it before)
- Create a course
- Now to the course as a student and click Contact teacher, send a message (This is the first message, not a reply)
- Now reply to that message from the teacher account
- Now reply again as the student
- Make sure the teacher received an email

## Pre-Merge Checklist
